examples/get_certificates_urls_example.py

```diff
@@ -0,0 +1,65 @@
+# Run with Python 3
+import requests
+
+# 1. Get your keys at https://stepic.org/oauth2/applications/ (client type = confidential,
+# authorization grant type = client credentials)
+client_id = "..."
+client_secret = "..."
+
+# 2. Get a token
+auth = requests.auth.HTTPBasicAuth(client_id, client_secret)
+resp = requests.post('https://stepic.org/oauth2/token/',
+                     data={'grant_type': 'client_credentials'},
+                     auth=auth
+                     )
+token = resp.json()['access_token']
+
+# 3. Call API (https://stepic.org/api/docs/) using this token.
+# Example:
+
+def get_certificates(page_number):
```

Comment:
This func only used in get_certificate_links?
So, `get_certificate_links` and `get_certificates` are redundant